en/contributing/build.md

```diff
@@ -245,9 +251,11 @@ You can also build the image yourself using the [Dockerfile](https://github.com/
 
 ## Build DroneCore Extensions {#dronecore_extensions}
 
-DroneCore can be extended with plugins and integration tests that are defined "out of tree". These are declared inside a parallel directory that is included into the DroneCore at compile time (by specifying `EXTERNAL_DIR` in the `make` command):
+DroneCore can be extended with plugins and integration tests that are defined "out of tree". These are declared inside a parallel directory that is included into the DroneCore at compile time (by specifying `EXTERNAL_DIR` in the `make` command).
+
+The commands to build and install DroneCore with an extension library are:
 ```
```

Comment:
Might be worth separating out and having separate build and install versions.

Reply:
I don't understand what you mean sorry.

Reply:
Don't worry about it. Mostly a note to me :-)